sympy/polys/rings.py

```diff
@@ -2532,6 +2566,135 @@ def compose(f, x, a=None):
 
         return poly
 
+    def primitive_wrt(self, x):
+        """
+        Returns the content and primitive part of a polynomial with respect to a
+        specified variable.
+
+        Examples
+        ========
+
+        >>> from sympy.polys import ring, ZZ
+        >>> R, x = ring("x", ZZ)
+        >>> p = 6*x**2 + 9*x**3
+        >>> p.primitive_wrt(x)
+        (9*x**3 + 6*x**2, 1)
```

Comment:
I've said this many times:

Doctest examples that show only trivial results do not help the reader to understand what the method does.

Reply:
The example needs to have more than one generator and neither of the content and primitive parts should be trivial. Also the example should be contrasted with the behaviour the `.primitive()` method to show how it differs.

Reply:
I am currently working on a task, and I cannot locate an example where the primitive part is not one at the moment.